src/main/scala/sigmastate/eval/Evaluation.scala

```diff
@@ -127,15 +132,75 @@ trait Evaluation extends RuntimeCosting { IR: IRContext =>
     case _ => false
   }
 
-  /** Checks if the function (Lambda node) given by the simbol `costF` contains only allowed operations
+  private val _allowedOpCodesInCosting = HashSet[OpCode](
+    Undefined,
```

Comment:
@greenhat is it really allowed?

Reply:
I suggest to remove this special code from the table and check it explicitly. This will also improve performance, because for all non-serializable nodes it will have table lookup time.

Reply:
Done.